doozerlib/rhcos.py

```diff
@@ -171,6 +171,9 @@ def __init__(self, runtime, pullspec_or_build_id: str, brew_arch: str):
     def __repr__(self):
         return f'RHCOSBuild:{self.brew_arch}:{self.build_id}'
 
+    def __repr__(self):
```

Comment:
@jupierce Was this added in error? Looks duplicate to the one above

Reply:
Yep - looks like a rebase artifact I miss. Removed. Thanks!